README.md

```diff
@@ -141,43 +138,22 @@ There are a number of arguments that are specific for each command. They can be
 
 #### mcluster specific args
 
-- `-i`, `--matrix` - TSV formatted distance matrix or parquet
+- `-i`, `--matrix` - TSV formatted distance matrix
 - `-d`, `--delimiter` - delimiter desired for nomenclature code [default="."]
 
 #### call specific args
 
-- `-d`, `--dists` - a 3 column file [query_id, ref_id, dist] in TSV or parquet format
-- `-r`, `--rclusters` - existing cluster file in TSV or parquet format
+- `-d`, `--dists` - a 3 column file [query_id, ref_id, dist] in TSV format
+- `-r`, `--rclusters` - existing cluster file in TSV format
 - `-j`, `--thresh_map` - Json file of [colname:threshold]
-- `-u`, `--outfmt` - output format for assignments [text (default), parquet]
 - `-l`, `--delimiter` - delimiter desired for nomenclature code [default="."]
 
-
 ## Configuration and Settings
 
 Thresholds must be configured when using GAS. These threshold must be determined manually through testing and establishment of practical criteria for each pathogen of interest. 
 
 For instance, in PulseNet Canada they have determined the use of '10,5,0' to be the threshold of choice for their pathogen surveillance program. [Publication on going]
 
-## Data Input/formats
-
-### Square distance matrix
```

Comment:
Square distance matrices are still used as part of `gas mcluster`, so I think we should still have a description of this format in the docs.

Was there no description of the three-column input type? Maybe we should have that included here as well (used for `gas call`).

Reply:
Oh that's right. I'll look over this again and re-add in a more clear way.

Reply:
I don't believe there was a description for the 3-column input, but maybe I'm misremembering.

Reply:
Thanks so much. I agree. Expanding can be in another PR.